### DISCUSSION TITLE
Web UI unreachable from internal subnet

### DISCUSSION THREAD

**Discussion by Bharmalej** [wg-easy/wg-easy#305] [Q&A]
Hello!

Can i accept to Web UI from internal subnet of other docker container? I bind address from docker.compose to wg-easy container, then try to open http://172.9.0.1:51821/ and have timeout, if i try open web ui from domain or external address -  all is ok.

Thank you for feedback!

**Comment by pheiduck** [comment]
Are you in the same network?
